girder_worker/core/cleanup.py

```diff
@@ -0,0 +1,5 @@
+from . import events
+
+
+def main():
+    events.trigger('cleanup')
```

Comment:
Are we keeping this around in case we want to trigger cleanup external ( crontab ) still?